extensions/cloud/portability-cloud-microsoft/src/main/java/org/dataportabilityproject/cloud/microsoft/cosmos/DataWrapper.java

```diff
@@ -0,0 +1,35 @@
+package org.dataportabilityproject.cloud.microsoft.cosmos;
+
+import com.microsoft.azure.storage.table.TableServiceEntity;
+
+/** Wraps Job data for serialization to Cosmos DB */
+public class DataWrapper extends TableServiceEntity {
```

Comment:
can this be package private?

Reply:
No because the Cosmos API implementations needs to reflectively instantiate types when deserializing.